nautobot/extras/migrations/0056_alter_configcontext_dynamic_groups.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2.16 on 2023-02-15 20:47
```

Comment:
What is the reason that this change required 2 migrations instead of one?

Reply:
Fixing this now. Will push shortly.